kclvm/api/src/testdata/override-file-bool.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Add more test cases about the unification stmt.

Reply:
fixed